test/test_torch.py

```diff
@@ -7659,7 +7661,11 @@ def test_data_ptr(getter):
 
     def test_to(self):
         self._test_to_with_layout(torch.strided)
-        self._test_to_with_layout(torch.sparse_csr)
+        is_cuda10_2_or_higher = (
+            (torch.version.cuda is not None)
+            and ([int(x) for x in torch.version.cuda.split(".")] >= [10, 2]))
+        if is_cuda10_2_or_higher:  # in cuda10_1 sparse_csr is beta
+            self._test_to_with_layout(torch.sparse_csr)
```

Comment:
Ideally this is deduplicated with the copy in test_cublas_config_nondeterministic_alert